kubernetes/resource_kubernetes_ingress_v1_test.go

```diff
@@ -310,19 +310,33 @@ func TestAccKubernetesIngressV1_multipleRulesDifferentHosts(t *testing.T) {
 	})
 }
 
-func testAccCheckKubernetesIngressV1ForceNew(old, new *networking.Ingress, wantNew bool) resource.TestCheckFunc {
```

Comment:
Not related to the original issue, just noticed that it is not in use, and since I touched this file, I cleaned it up a bit.